0001-typedb/01-start/03-quickstart.md

```diff
@@ -0,0 +1,673 @@
+---
+pageTitle: Quickstart guide
+keywords: getting started, typedb, typeql, tutorial, quickstart, overview
+longTailKeywords: get started with typedb, typedb tutorial, typedb quickstart, learn typedb
+summary: Learn how to create a TypeDB database, load schema and data, perform queries.
+toc: false
+---
+
+# Quickstart guide
+
+## Overview
+
+This Quickstart guide goes through the step-by-step process of creating and interacting with a TypeDB database designed 
+for an [Identity and Access Management](https://en.wikipedia.org/wiki/Identity_management) solution that will be 
+described [later](04-iam-schema.md).
+
+<div class="note">
+[Important]
+We highly recommend completing this guide. Its goal is to prepare an environment for TypeDB exploration and development.
+The resulting database (schema and data) will be needed for example queries throughout the documentation.
+</div>
+
+## Prerequisites
+
+This Quickstart guide takes advantage of TypeDB Studio, a desktop GUI tool for interacting with TypeDB. To proceed, 
+please install both TypeDB and TypeDB Studio:
+
+- [Install TypeDB](02-installation.md)
+- [Install TypeDB Studio](../../02-clients/01-studio.md#get-typedb-studio)
+
+IMPORTANT:
+<div class="note">
+[Important]
+It’s recommended to use TypeDB and TypeDB Studio of the same version number in order to ensure compatibility. If 
+TypeDB Studio doesn't have a version similar to the TypeDB release — use the closest lower version instead.
+</div>
+
+## Initialize database
+
+### Start TypeDB
+
+Run the following command in a terminal:
+
+<!-- test-ignore -->
+```bash
+typedb server
+```
+
+After showing the TypeDB ASCII logo and the bootup completion time, TypeDB is ready for connections.
+
+![TypeDB server bootup message](../../images/quickstart/quickstart-launched-typedb.png)
+
+### Connect to TypeDB
+
+Launch TypeDB Studio, then:
+
+1. Click the [**Connect to TypeDB**] button (right side of the toolbar).
+2. Enter `localhost:1729` in the [**Address**] field.
+3. Click the [**Connect**] button (the dialog will close after a successful connection).
+
+![Connection Manager Connected](../../images/studio/connection-interface-connected.png)
+
+### Create a project
+
+TypeDB Studio projects allow you to organize and save related queries for future reuse.
+
+![Project Interface empty](../../images/quickstart/studio-projects-empty.png)
+
+To create a new project:
+
+1. Click the [**Open Project**] button in the [**Project**] panel (upper left).
+2. Choose a directory for the project files.
+3. Click the [**Open**] button.
+
+The [**Project**] panel will now display the root directory and a nested hidden directory for unsaved files.
```

Comment:
Might be worth saying the 'hidden' directory is called '.typedb-studio'

Reply:
I don't see any reason to do that. We don't use the hidden directory for anything here. I only mentioned it to add some context to users on what they see on screen. We don't need its name to do that.